catalyst_voices/packages/internal/catalyst_voices_models/lib/src/json_converters.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:json_annotation/json_annotation.dart';
+
+final class DurationConverter implements JsonConverter<Duration, int> {
```

Comment:
It's a bit unusual we have it here, normally this belongs to data/repositories layer, `models` should be "free" of serialization/deserialization logic.

Reply:
given how `json_serializable` works do you see any ways for us using it as serialisation lib? I don't see any ways for doing it but duplicating all of models which should be serialised. 

we kind of started doing it in #1363 too

Reply:
Firstly, we could move this DurationConverter somewhere to catalyst_voices_shared to don't bloat models with this kind of code? Then there's the issue whether we want to have separately DTO's with `fromJson`/`toJson` and separately models. imho don't like the duplication. Lets just find a better place for this class